packages/devextreme/js/__internal/scheduler/__tests__/__mock__/model/popup.ts

```diff
@@ -0,0 +1,136 @@
+export class PopupModel {
+  element: HTMLDivElement;
+
+  constructor(element: HTMLDivElement) {
+    this.element = element;
+  }
+
+  getLabelIdByText = (labelText: string): string => {
+    const labels = Array.from(this.element.querySelectorAll('label'));
+
+    const label = labels.find((l) => l?.textContent?.trim()?.startsWith(labelText));
```

Comment:
nitpick: one letter variables not easy to select by mouse or touchpad. In my perspective, better just use general word like `item`

No need to fix. Just thoughts